diff --git a/node_modules/electron-packager/src/cli.js b/node_modules/electron-packager/src/cli.js
index 57dae02..f158c72 100644
--- a/node_modules/electron-packager/src/cli.js
+++ b/node_modules/electron-packager/src/cli.js
@@ -67,6 +67,9 @@ module.exports = {
     if (args.osxSign === 'true') {
       warning('--osx-sign does not take any arguments, it only has sub-properties (see --help)')
       args.osxSign = true
+    } else if (typeof args.osxSign === 'object' && !Array.isArray(args.osxSign)) {
+      // Keep kebab case of sub properties
+      args.osxSign = args['osx-sign']
     }
 
     if (args.osxNotarize) {
